.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "FV3"]
   path = FV3
-  url = https://github.com/NOAA-GSL/fv3atm
-  branch = RRFS_dev
+  url = https://github.com/haiqinli/fv3atm
+  branch = RRFS_dev_sd
```

Comment:
This should not point to your own branch and repository.

Reply:
It is updated to point to the NOAA-GSL repository and RRFS_dev branch.

Reply:
This is a mistake. It should not point to NOAA-GSL until the PR is merged into NOAA-GSL. Sorry for confusion.

Reply:
@haiqinli Canyou switch this back? thanks.

Reply:
@hu5970 It would be better if you made your own PRs at the ufs-weather-model and fv3atm level, pointing to @haiqinli's  underlying ccpp-physics.

Reply:
There is nothing wrong with pointing to your own repository. Many people do that, and it is corrected during the commit process.

What's most important is that the hashes be up-to-date, which they aren't right now. The FV3/ccpp/physics differs from the top of the ccpp-physics PR